### ISSUE TITLE
Document the new Service & Event APIs

### ISSUE THREAD

**Issue by lokka30** [lokka30/Treasury#146]
I'll need to document the new service & event APIs before we release v1.1.0.

**Comment by lokka30** [comment]
On hold: waiting for the resolution of #147.

**Comment by MrIvanPlays** [comment]
Already documented

**Comment by lokka30** [comment]
Awesome. Waiting on the result of #147 to close this.

**Comment by MrIvanPlays** [comment]
Well if #147 turns out to be done then the only thing which will need update is the javadocs links on the wiki page. Nothing else
